test/Generator/MethodGeneratorTest.php

```diff
@@ -74,6 +74,38 @@ public function testMethodParameterMutator()
         $methodGenerator->setParameter(new stdClass());
     }
 
+    public function testMethodParameterSetPosition()
```

Comment:
We should have two separate test methods:

 * one verifying `setParameter()`
 * one verifying `setParameters()`